stock_serial_batch_receive/models/stock_move.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import models, fields
+
+
+class StockMove(models.Model):
+
+    _inherit = 'stock.move'
+
+    picking_type_create_lots = fields.Boolean(
+        related='picking_id.picking_type_id.use_create_lots',
+    )
+    first_number = fields.Char(
```

Comment:
Do we really need the first_number here ? IMO it should only be defined on the wizard.